operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md

```diff
@@ -334,6 +334,9 @@ Some systems are configured with lazy mounts that do not have this requirement f
     Example output:
 
     ```text
+    cray-spire-jwks-6b97457548-gc7td    2/3  CrashLoopBackOff   9    23h   10.44.0.117  ncn-w002 <none>   <none>
```

Comment:
The tail end of these three cray-spire-jwks pod names will be different from the spire-jwks pods.  Also the IP addresses should be different in this example.

Reply:
Updated.